os/board/stm32l4r9ai-disco/include/stm32l4r9i_discovery.h

```diff
@@ -359,6 +359,8 @@ extern void stm32_touch_printf_coord(void);
 
 extern void stm32_dsi_refresh(void);
 extern void up_hal_timer_initialize(void);
+
+extern void stm32l4_flash_initialize(void);

```

Comment:
May I know reason why you add this?

Reply:
I added it to call this function in stm32_boot.c